update container-runtime.md (#42502)
kubernetes/website: content/en/docs/reference/glossary/container-runtime.md

- fundamental
- workload
---
 A fundamental component that empowers Kubernetes to run containers effectively.

>  Its responsibility lies in managing the execution and lifecycle of containers within the Kubernetes environment.

Reviewer: Making this a little more idiomatic,

```suggestion
 It is responsible for managing the execution and lifecycle of containers within the Kubernetes environment.
```

>  Kubernetes doesn’t know how to run containers it relies on the container runtime component. 
>  The container runtime is a fundamental component that is responsible for managing the execution and lifecycle of containers.

Reviewer 2: ```suggestion
 A fundamental component that empowers Kubernetes to run containers effectively.
 Its responsibility lies in managing the execution and lifecycle of containers within the Kubernetes environment.
```
Only for your reference.
